src/lib/contexts/TalentContext.tsx

```
@@ -0,0 +1,274 @@
+        // Rating filter
+        if (talent.rating < state.filters.rating) {
+          return false
+        }
+
+        // Availability filter - check if talent has availability info
+        if (state.filters.availability && (!talent.availability || talent.availability === "")) {
+          return false
+        }
+
+        return true
+      })
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)